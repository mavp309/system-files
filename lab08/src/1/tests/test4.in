0.00001 0.99999
-7.77777 2.22222
