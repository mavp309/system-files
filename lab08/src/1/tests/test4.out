Sum: -7.7778+3.2222j
Difference: 7.7778+-1.2222j
Product: -2.2223+-7.7777j
Division: 0.0340+-0.1189j
